lib/dslash_domain_wall_4d_fused_m5.hpp

```diff
@@ -111,19 +103,17 @@ namespace quda
       default: errorQuda("Unexpected Dslash5Type %d", static_cast<int>(Arg::dslash5_type));
       }
 
-      return flops_ + Dslash::flops();
+      return in.size() * flops_ + Dslash::flops();
     }
 
     long long bytes() const override
     {
       if (Arg::dslash5_type == Dslash5Type::M5_INV_MOBIUS_M5_INV_DAG) {
-        return arg.y.Bytes() + Dslash::bytes();
+        return in.size() * arg.y[0].Bytes() + Dslash::bytes();
```

Comment:
Remove the `[0]`

Reply:
[2a5acd0](https://github.com/lattice/quda/pull/1465/commits/2a5acd0c72bb6aa8de0932f4cf4c7abae8661791)